packages/team-management/src/components/TeamsView/Table.tsx

```diff
@@ -34,7 +34,6 @@ const Table: React.FC<Props> = (props: Props) => {
     },
     {
       title: 'Actions',
-      dataIndex: 'operation',
```

Comment:
I feel this would have been accompanied by a corresponding adjustment of a test case. I find it easier creating a  util function to get the table columns which is testable, such is done in some packages e.g [User management](https://github.com/OpenSRP/web/blob/master/packages/keycloak-user-management/src/components/UserList/utils.tsx#L17)

Reply:
@kelvin-muchiri this is unused code that is being removed, I think the test here would be to make sure regression does not happen. that being said I think u did raise a proper concern.